### ISSUE TITLE
CosyVoice  flow2  training from scratch

### ISSUE THREAD

**Issue by JohnHerry** [FunAudioLLM/CosyVoice#1122]
@aluminumbox 
Hi, I am training the flow2 on the dev/lyuxiang.lx  branch.   training the basemodel.  so I changed the experiments/libritts/cosyvoice2/conf/cosyvoice2.yaml   from SFT training config into base model config where I changed the learning rate, scheduler,  warmup_steps, and max_frames_in_batch.  the max_frames_in_batch valued from 2000 into 8000.
     but when traning run,  here comes the warnings like:  
`get chunk_masks all false at some timestep, force set to true, make sure they are masked in futuer computation!`   
and  a lot of attention_mask generated in the `CausalAttnProcessor2_0`  line  `if attention_mask is not None:  attention_mask = attention_mask.unsqueeze(dim=1).repeat(1, attn.heads, 1, 1)`  ,  then  OOM and crush.   what happend  then?

**Comment by JohnHerry** [comment]
看了一下 开源的训练配置，貌似只是支持 chunk_size =2 秒的流式模型的训练，并不是论文中的 四种 chunk-aware 均匀采样策略的训练。为什么这么做呢？ 是训练固定chunk_size的模型，比论文中的四种chunk-ware一起训练，效果更好吗【在特定chunk=2秒的推理场景下】？

**Comment by aluminumbox** [comment]
update the code, flow matching training is updated yesterday

**Comment by aluminumbox** [comment]
> 看了一下 开源的训练配置，貌似只是支持 chunk_size =2 秒的流式模型的训练，并不是论文中的 四种 chunk-aware 均匀采样策略的训练。为什么这么做呢？ 是训练固定chunk_size的模型，比论文中的四种chunk-ware一起训练，效果更好吗【在特定chunk=2秒的推理场景下】？

如果确定只做流式推理，那么只做流式训练是最好的
